tests/infra/ca_certs.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
infra/ is for infrastructure used by tests, not tests themselves, why is this moved here?

Reply:
It's a duplicate and I've no idea how it ended up there. Gone now.